examples/achromatic_spectrometer/README.rst

```diff
@@ -0,0 +1,78 @@
+.. _examples-fodo:
+
+FODO Cell
```

Comment:
This file needs an update :)

It als oneeds to be linked in `docs/source/usage/examples.rst`.

Reply:
Just a placeholder for now.